### DISCUSSION TITLE
UpdateHub v2.4.0

### DISCUSSION THREAD

**Discussion by onyx-nxt** [NexovaDev/UpdateHub#1329] [Announcements]
## Version
`2.4.0`

# Changelog
- Improve UpdateHub auto-check update interval
- Added scroll to bottom button if page is large
- Fix minor styling
- Improve app startup cycle
- Use native startup key method
- Improve post-install performance
- Improve search matching
- Remove duplicate items in the recently updated list
- Patch fix Application User Model ID (this fixes the notification titles and such)
- And more bug fixes!

# How to update
Click the Update Available button in the sidebar. (if you don't see it, force close the app by pressing the shift key, clicking the X icon in the windows title bar, and then relaunching the app.)

<img src="https://github.com/NexovaDev/UpdateHub/assets/71038229/d190f605-897d-4e6f-b147-fcccb07bdfe2" width="584" /> \
Then wait for the update file to download, and it will install automatically. You now have the latest version!

### VirusTotal scan
This app is not a virus, trust me. If you want, here's a [VirusTotal scan](https://www.virustotal.com/gui/file/f1e44905005cd60513ac9e5fdc96e6ef33bace1f1f57b0ed9138320f09b9c2b6 "VirusTotal Scan"). As you can see, no anti-virus software has flagged it as a virus. The executable below's SHA-256 Hash is `F1E44905005CD60513AC9E5FDC96E6EF33BACE1F1F57B0ED9138320F09B9C2B6`, which matches the VirusTotal scan.

### Remember, if you encounter any bugs in UpdateHub, report them so we can fix them!

The one-click installer is attached below.

### SHA256
`F1E44905005CD60513AC9E5FDC96E6EF33BACE1F1F57B0ED9138320F09B9C2B6`

<hr /><em>This discussion was created from the release <a href='https://github.com/NexovaDev/UpdateHub/releases/tag/v2.4.0'>UpdateHub v2.4.0</a>.</em>

**Comment by farag2** [comment]


**Reply by marjuanm** [reply]
Yes, I see that this project seems to be dead. A shame.

**Comment by rdeckard2019** [comment]
Is this really dead? Truly a shame if it is since it appeared to have potential.
